README.md

```diff
@@ -87,28 +87,27 @@ This is an static landing page and so, I (cynthia, who is writing this nice expl
 ---
 
 #CONTRIBUTE
-## Do you wanna contribute providing translation for the new openSUSE website?
+## Do you want to contribute by providing translations for the new openSUSE website?
 If yes is your answer, then you will make thousands of users happy to have the openSUSE landing page in their language.
 
-The openSUSE translations are hosted in subversion on http://svn.opensuse.org - the template for it is opensuse-landing-page.pot. You need to update
-the $LANG/po/opensuse-landing-page.$LANG.po within subversion. 
+The openSUSE translations are provided by https://l10n.opensuse.org/. You can log in with your openSUSE account there.
```

Comment:
l10n has links for contact and documentation, that's way more than the previous mention of svn.o.o ever had.